app/scripts/components/analysis/page-hero-analysis.js

```diff
@@ -173,5 +250,54 @@ PageHeroAnalysis.propTypes = {
   description: T.string,
   renderAlphaBlock: T.func,
   renderBetaBlock: T.func,
-  isHidden: T.bool
+  isHidden: T.bool,
+  isResults: T.bool
 };
+
+function useIsStuck(threshold) {
+  const [isStuck, setStuck] = useState(window.scrollY > threshold);
+
+  useEffect(() => {
+    // Check for the observer pixel.
+    // https://mediatemple.net/blog/web-development-tech/using-intersectionobserver-to-check-if-page-scrolled-past-certain-point-2/
+    const pixel = document.createElement('div');
+    pixel.id = 'page-hero-pixel';
```

Comment:
Would be good to store `page-hero-pixel` in a constant.

Reply:
good call!